doc/maintaining-V8.md

```diff
@@ -0,0 +1,275 @@
+   </td>
+   <td>2016-04-01
+   </td>
+   <td>2019-04-01
+   </td>
+   <td>5.1
+   </td>
+   <td>2016-05-31
+   </td>
+   <td>2016-06-26
+   </td>
+  </tr>
+  <tr>
+   <td>Node v7.x
+   </td>
+   <td>2016-10-01
+   </td>
+   <td>2017-04-01
+   </td>
+   <td>5.4
+   </td>
+   <td>In beta
+   </td>
+   <td>~2016-12-01
+   </td>
+  </tr>
+  <tr>
+   <td>master
+   </td>
+   <td>N/A
+   </td>
+   <td>N/A
+   </td>
+   <td>5.4
+   </td>
+   <td>In beta
+   </td>
+   <td>~2016-12-01
+   </td>
+  </tr>
+</table>
+
+
+The versions of V8 used in Node v4.x and v6.x have already been abandoned by upstream V8. However, Node.js needs to continue supporting these branches for many months (Current branches) or several years (LTS branches).
+
+# Maintenance Process
+
+Once a bug in Node.js has been identified to be caused by V8, the first step is to identify the versions of Node and V8 affected. The bug may be present in multiple different locations, each of which follows a slightly different process.
+
+* Unfixed bugs. If the bug exists in the V8 master branch.
+* Fixed, but needs backport. The bug may need porting to one or more branches.
+    * Backporting to active branches.
+    * Backporting to abandoned branches.
+* Backports identified by the V8 team.
+
+## Unfixed Upstream Bugs
+
+If the bug can be reproduced on the `vee-eight-lkgr` branch, Chromium canary, or V8 tip-of-tree, and the test case is valid, then the bug needs to be fixed upstream first.
+
+* Start by opening a bug upstream using this template: https://bugs.chromium.org/p/v8/issues/entry?template=Node.js%20upstream%20bug.
+* Make sure to include a link to the corresponding Node.js issue (if one exists)
+* If the fix is simple enough, you may fix it yourself; [contributions](https://github.com/v8/v8/wiki/Contributing) are welcome.
+* V8's build waterfall tests your change.
+* Once the bug is fixed it may still need backporting, if it exists in other V8 branches that are still active or are branches that Node cares about. Follow the process for backporting below.
+
+## Backporting to Active Branches
+
+If the bug exists in any of the active V8 branches, we may need to get the fix backported. At any given time there are[ two active branches](https://build.chromium.org/p/client.v8.branches/console) (beta and stable) in addition to master. The following steps are needed to backport the fix:
+
+* Identify which version of V8 the bug was fixed in.
+* Identify if any active V8 branches still contain the bug:
+* A tracking bug is needed to request a backport.
+    * If there isn't already a V8 bug tracking the fix, open a new merge request bug using this [Node.js specific template](https://bugs.chromium.org/p/v8/issues/entry?template=Node.js%20merge%20request).
+    * If a bug already exists
+        * Add a reference to the GitHub issue.
+        * Attach *merge-request-x.x* labels to the bug for any active branches that still contain the bug. (e.g. merge-request-5.3, merge-request-5.4)
```

Comment:
I will adjust the doc to suggest that folks on @nodejs/v8 can make these edits on the V8 issue tracker.